website/docs/flink_configuration.md

```diff
@@ -3,115 +3,179 @@ title: Flink Setup
 toc: true
 ---
 
-## Global Configurations
-When using Flink, you can set some global configurations in `$FLINK_HOME/conf/flink-conf.yaml`
+[Apache Flink](https://flink.apache.org/what-is-flink/flink-architecture/) is a powerful streaming-batch integrated engine that provides a stream processing framework. Flink can process events at an incredible speed with low latency. Along with Hudi, users can use streaming ingestion like with Kafka; streaming consumption like with Kafka; and also perform batch workloads like bulk ingest, snapshot queries and incremental queries. 
 
```

Comment:
> users can use streaming ingestion like with Kafka; streaming consumption like with Kafka

A littile verbose here, maybe we can just merge the ingestion and consumption thing like in one sentence.

Reply:
+1

Reply:
Along with Hudi, you can use streaming ingestion and consumption with sources like Kafka; and also perform batch workloads like bulk ingest, snapshot queries and incremental queries.

Reply:
cc @danny0405  more simplified- wdyt